src/components/report-viewer/report-viewer.tsx

```diff
@@ -12,12 +12,23 @@ import { VirtualizedTreeview } from './virtualized-treeview';
 import { ReportItem } from './treeview-item';
 import { Report, ReportLog, ReportTree, ReportType, SelectedReportLog, SeverityLevel } from 'utils/report/report.type';
 import { GLOBAL_REPORT_NODE_LABEL } from '../../utils/report/report.constant';
+import ComputingType from 'components/computing-status/computing-type';
 
-type ReportViewerProps = { report: Report; reportType: string; severities: SeverityLevel[] | undefined };
+type ReportViewerProps = {
+    report: Report;
+    reportType: ComputingType;
```

Comment:
Won't this break once `ReportViewerTab` is migrated to tsx since it feeds `COMPUTING_AND_NETWORK_MODIFICATION_TYPE.NETWORK_MODIFICATION` to it ?

Reply:
Good catch